src/app/cases/creator/case-creator-submit.component.ts

```diff
@@ -44,6 +47,29 @@ export class CaseCreatorSubmitComponent implements OnInit {
     return (sanitizedEditForm: CaseEventData) => this.casesService.validateCase(this.jurisdictionId, this.caseTypeId, sanitizedEditForm);
   }
 
+  private canSaveDraft(): boolean {
```

Comment:
is it possible to move this method on 
`CaseEventTrigger`

Reply:
you might have problem in making that work because of the way we deserialise objects, but let's give it a try?

Reply:
also, how about we call this property 
`eventAllowsDraft` 
? 
can_save_draft could have various interpretations.

Reply:
Have a look at the profile.model.ts, which has a isSolicitor method. 
And the new way we deserialise it in profile.service.ts (i.e. using the library class-transformer)